### PR TITLE
Enhance Announcements Model 

### PR DESCRIPTION
**Pull request by kracekumar**
### Issue Reference
This PR addresses the Issue : Fixes #

### Summarize
- Remove link field, add image, attachement, is_pinned, description fields in announcement model.
- Display pinned announcement in the announcement list page.
- Add `All` as a district.

> Also, mention the key points if any to look into while code reviews

### CODE REVIEW

**Comment by biswaz** [comment]
@ashwinrajeev Urgent merge required

**Comment by biswaz** [comment]
Don't merge. This will fail

**Comment by biswaz** [comment]
Heroku wont store media uploads. It will disappear. Please test with S3 as requested

**Comment by kracekumar** [comment]
Sure, will use s3

**Comment by ashwinrajeev** [comment]
I am getting an error while adding announcements from admin view.

`null value in column "name" violates not-null constraint`

**Comment by biswaz** [comment]
Log please

**Comment by kracekumar** [comment]
@biswaz 
```

Request Method: | POST
-- | --
http://localhost:8000/admin/mainapp/announcements/add/
2.1
S3ResponseError
S3ResponseError: 400 Bad Request
/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python3.5/site-packages/boto/s3/bucket.py in _get_key_internal, line 232
/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/bin/python3
3.5.2
['/home/krace/code/rescuekerala',  '/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python35.zip',  '/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python3.5',  '/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python3.5/plat-x86_64-linux-gnu',  '/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python3.5/lib-dynload',  '/usr/lib/python3.5',  '/usr/lib/python3.5/plat-x86_64-linux-gnu',  '/home/krace/.local/share/virtualenvs/rescuekerala-O6kS_LhK/lib/python3.5/site-packages']
Sat, 18 Aug 2018 19:16:04 +0530


```

**Comment by biswaz** [comment]
@kracekumar What does slack say about this?

File: mainapp/models.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
     ('tvm','Thiruvananthapuram - തിരുവനന്തപുരം'),
     ('tcr','Thrissur - തൃശ്ശൂർ'),
     ('wnd','Wayanad - വയനാട്'),
+    ('All', 'All')
```
[the remark this answers is not in the document]
Comment (biswaz) [reply mainapp/models.py]: Yep

**Comment by biswaz** [comment]
I'm redoing this manaully, it needs rebase

**Comment by kracekumar** [comment]
New PR: https://github.com/IEEEKeralaSection/rescuekerala/pull/610/

**Pull request closed without merging**